.gitignore

```diff
@@ -1 +1,5 @@
-.azure
\ No newline at end of file
+.azure
```

Comment:
I'm getting this error when I run azd provision
![image](https://github.com/Azure-Samples/demystifying-rag-dotnet/assets/2163001/3e3043f6-44ec-4dde-aaec-18e004cd17df)

Reply:
Since the default value of `OPENAI_HOST` has been removed, we need to set it before executing `azd provision`. You can make it pass by executing `azd env set OPENAI_HOST azure/openai`.

Reply:
Let's default to azure.